test/functional/pdo_sqlsrv/pdo_param_output_variants.phpt

```diff
@@ -1,75 +1,82 @@
 --TEST--
 Test parametrized insert and sql_variant as an output parameter.
 --DESCRIPTION--
-Since output param is not supported for sql_variant columns, this test verifies a proper error message is returned  
+Since output param is not supported for sql_variant columns, this test verifies a proper error message is returned
 --FILE--
 ﻿<?php
-include 'MsCommon.inc';
+require_once("MsCommon_mid-refactor.inc");
 
-function TestReverse($conn)
+function testReverse($conn)
 {
-    $procName = GetTempProcName('sqlReverse');
+    $procName = getProcName('sqlReverse');
 
-    try
-    {
+    try {
         $spCode = "CREATE PROC [$procName] @string AS SQL_VARIANT OUTPUT as SELECT @string = REVERSE(CAST(@string AS varchar(30)))";
-
-        $stmt = $conn->exec($spCode);
-    }
-    catch (Exception $e)
-    {
+        $conn->exec($spCode);
+    } catch (Exception $e) {
         echo "Failed to create the reverse procedure\n";
         echo $e->getMessage();
-    }        
-    
-    try
-    {
-        $stmt = $conn->prepare("{ CALL [$procName] (?) }");  
+    }
+
+    try {
+        $stmt = $conn->prepare("{ CALL [$procName] (?) }");
         $string = "123456789";
-        $stmt->bindParam(1, $string, PDO::PARAM_STR, 30); 
+        $stmt->bindParam(1, $string, PDO::PARAM_STR  | PDO::PARAM_INPUT_OUTPUT, 30);
         $stmt->execute();
-        echo "Does REVERSE work? $string \n";  
-    }
-    catch (Exception $e)
-    {
+        // Connection with Column Encryption enabled works for non encrypted SQL_VARIANT
+        // Since SQLDescribeParam is called
+        if (isColEncrypted() && $string === "987654321") {
+            echo "Test input output parameter with SQL_VARIANT successfully.\n";
+            
```

Comment:
successfully _done_ or _completed_

Reply:
More code seems to be involved if I do it that way. I'll leave it the way it is now.

Reply:
Fine. Please just fix the grammar in the output message :)